ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/subversion/SubversionCredentialsDialog.java

```diff
@@ -8,33 +8,21 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.plugin.svn.ide.askcredentials;
+package org.eclipse.che.ide.api.subversion;
```

Comment:
It is used in `ProjectImporter` that is in `che-core-ide-ap`. I can not add `plugin-svn` dependency there.